src/web/components/Column/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Column({ children, size }) {
+  let classes = 'column';
+  if (size > 0 && size < 11) {
+    classes += ` is-${size}`;
+  }
+
+  return <div className={classes}>{children}</div>;
+}
+
+Column.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.string,
```

Comment:
We'll never accept only `string` as children. Only elements or array of elements